### ISSUE TITLE
v3.6.0 allows Slim 5.0.0, which has a major issue with Rails (fixed on slim main but unreleased)

### ISSUE THREAD

**Issue by tagliala** [slim-template/slim-rails#187]
Hello,

dependabot found an issue while trying to update from 3.5.0 to 3.6.0:

https://github.com/diowa/ruby3-rails7-bootstrap-heroku/pull/383

https://github.com/diowa/ruby3-rails7-bootstrap-heroku/actions/runs/4170639779/jobs/7219800545

```
Run bundle exec rake lint
  bundle exec rake lint
  shell: /usr/bin/bash -e {0}
rake aborted!
NameError: uninitialized constant Slim::Rails::Railtie
```

https://github.com/diowa/ruby3-rails7-bootstrap-heroku/actions/runs/4170639772/jobs/7219800543

```
Run bundle exec rails db:prepare
rails aborted!
NameError: uninitialized constant Slim::Rails::Railtie
```

**Comment by apannetrat** [comment]
Same problem here. Even running `rails c` after an upgrade to 3.6.0 gives:
```
.rvm/gems/ruby-2.7.6/gems/slim-5.0.0/lib/slim/railtie.rb:2:in `<module:Slim>': uninitialized constant Slim::Rails::Railtie (NameError)
```

**Comment by tagliala** [comment]
Ah sorry,

the braking change is related to Slim 5.0, which is being update together with 3.6.0

```
/Users/geremia/.rvm/gems/ruby-3.2.0/gems/slim-5.0.0/lib/slim/railtie.rb:2:in `<module:Slim>': uninitialized constant Slim::Rails::Railtie (NameError)

  class Railtie < Rails::Railtie
```

I'm leaving this open, but feel free to close or move to the proper repo

**Comment by mkhairi** [comment]
https://github.com/slim-template/slim/pull/904

**Comment by januszm** [comment]
It's unfortunately a result of https://github.com/slim-template/slim-rails/pull/185 which apparently wasn't fully tested with Rails and SLIM 5. We can yank 3.6.0 and release proper support for SLIM 5 when SLIM is fixed, as 3.6.1. A quick workaround for apps, until `slim` 5.0.1+ is released, is to lock `gem "slim", "< 5.0.0", "> 5.0.0"` in their Gemfile

Personally, I'd prefer to wait until SLIM is resolved rather than yank this.

**Comment by tagliala** [comment]
> We can yank 3.6.0

Hi, I know that yanking is not ideal, but I was about to suggesting the same thing

Should a potential 3.6.1 add `!= 5.0.0` to the slim version constraints even after release of 5.0.1?

**Comment by januszm** [comment]
I'll release 3.6.1 now with this fix in gemspec

**Comment by tagliala** [comment]
Thanks, closing as fixed in 3.6.1
